implementations/rust/ockam/src/address.rs

```diff
@@ -3,17 +3,33 @@ use alloc::string::{String, ToString};
 use core::fmt::{Display, Formatter};
 use core::hash::{Hash, Hasher};
 
+#[derive(Eq, PartialEq, Clone, Debug)]
+pub enum AddressType {
+    Worker = 0,
+    Undefined = 255,
+}
+
```

Comment:
I think it would be nice if we can avoid a global list of address types.
Each transport knows the number (representing its address type) it has picked but there is no global list.